src/App.jsx

```diff
@@ -1,23 +1,27 @@
 import React, { useState, useEffect } from 'react';
+import { ColorRing } from 'react-loader-spinner';
 import data from './data.json';
 
 import Header from './Header';
 import ToDoList from './ToDoList';
 import ToDoForm from './ToDoForm';
 
+async function fetchData(setToDoList) {
```

Comment:
also, since you are fetching the todo list, let's move this fetching function into the TodoList component